### ISSUE TITLE
"Failed to load" error when syncing to other devices.

### ISSUE THREAD

**Issue by h4rrydog** [daledesilva/obsidian_ink#72]
I am using the Ink plugin with Obsidian Sync on MacOS, Android and iPadOS. 

Everything works fine on MacOS - I can write and draw and everything appears normal.

The plugin syncs through Obsidian Sync to my phone and iPad, but the Ink plugin won't enable on both of these devices.

When I try to enable the plugin on my phone and tablet, I get the error 'Failed to load plugin "ink"'.

The expected behavior is for it to work across platforms. From other issues in the issue queue, it would appear that it works for other people on Android and iPadOS (presumably, as people talk about adding features with Apple Pencil). But I can't get it to work for me on Android and iPadOS.

- Macbook Pro M3, MacOS Sonoma 14.5
- Pixel 5, Android 14
- iPad Mini 5, iPadOS 17.5.1

Thanks for your hard work on this addon. I'm also a fellow OneNote refugee, so really looking to make this work. :)

**Comment by h4rrydog** [comment]
I managed to make it work by installing the plugin on each device.

Obsidian Sync doesn't seem to sync the add-on correctly, for some reason.

**Comment by daledesilva** [comment]
Oh, that's good to know. Thanks for following up.

Out of curiosity, did you try closing and relaunching obsidian after sync? Perhaps it synced by need a restart to initiate properly.

**Comment by h4rrydog** [comment]
Yes, I did that an all the devices... multiple times without success. :( 
But I can't rule out that it is a user error on my part.
